Neos.Neos/Classes/Controller/Backend/MenuHelper.php

```diff
@@ -81,34 +93,38 @@ public function buildSiteList(ControllerContext $controllerContext): array
             return [];
         }
 
+        $context = $this->contextFactory->create();
         $domainsFound = false;
         $sites = [];
         foreach ($this->siteRepository->findOnline() as $site) {
-            $uri = null;
-            $active = false;
-            /** @var $site Site */
-            if ($site->hasActiveDomains()) {
-                $activeHostPatterns = $site->getActiveDomains()->map(static function ($domain) {
-                    return $domain->getHostname();
-                })->toArray();
-
-                $active = in_array($requestUriHost, $activeHostPatterns, true);
-
-                if ($active) {
-                    $uri = $contentModule['uri'];
-                } else {
-                    $uri = $controllerContext->getUriBuilder()->reset()->uriFor('switchSite', ['site' => $site], 'Backend\Backend', 'Neos.Neos');
+            $node = $context->getNode(SiteService::SITES_ROOT_PATH . '/' . $site->getNodeName());
```

Comment:
```suggestion
            $node = $context->getNode(\Neos\ContentRepository\Domain\Utility\NodePaths::addNodePathSegment(SiteService::SITES_ROOT_PATH, $site->getNodeName()));
```

i just noticed, that in all other places we use this helper instead of manual string concatenation.

but otherwise fine by reading ^^